acceptance-tests/src/test/java/tech/pegasys/eth2signer/dsl/signer/runner/Eth2SignerRunner.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.eth2signer.dsl.signer.runner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import tech.pegasys.eth2signer.dsl.signer.SignerConfiguration;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
+
+import com.google.common.io.MoreFiles;
+import com.google.common.io.RecursiveDeleteOption;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.awaitility.Awaitility;
+
+public abstract class Eth2SignerRunner {
+
+  private static final Logger LOG = LogManager.getLogger();
+
+  private final SignerConfiguration signerConfig;
+  private final Path dataPath;
+  private final Properties portsProperties;
+
+  private static final String PORTS_FILENAME = "eth2signer.ports";
+  private static final String HTTP_PORT_KEY = "http-port";
+
+  public static Eth2SignerRunner createRunner(final SignerConfiguration signerConfig) {
+    if (Boolean.getBoolean("acctests.runEth2SignerAsProcess")) {
+      return new Eth2SignerProcessRunner(signerConfig);
```

Comment:
Think would be useful to log which runner we are using

Reply:
done